$NetBSD: patch-af,v 1.4 2013/06/12 01:57:24 dholland Exp $

- use standard headers
- the type of main() does not involve size_t

--- tools/depbuild.cpp.orig	2006-12-27 11:04:05.000000000 +0000
+++ tools/depbuild.cpp
@@ -24,6 +24,8 @@ This program generates dependencies for 
 #include <iostream>
 #include <string>
 #include <cstdio>
+#include <cstdlib>
+#include <unistd.h>
 using namespace std;
 
 #include "fileutil.h"
@@ -183,7 +185,7 @@ void dependency_calculate(const char *fi
   }
 }
 
-int main(size_t argc, const char *const *const argv)
+int main(int argc, const char *const *const argv)
 {
   if (argc < 5)
   {
@@ -209,7 +211,7 @@ int main(size_t argc, const char *const 
     else
     {
       struct stat unused;
-      for (size_t i = 5; i < argc; i++)
+      for (int i = 5; i < argc; i++)
       {
         dependency_calculate(argv[i], unused);
       }
